lametro/events.py

```diff
@@ -90,25 +86,30 @@ def api_events(self, *args, **kwargs):
     def _merge_events(self, events):
         english_events = []
         spanish_events = []
-        
+
         for event, web_event in events:
             if event.is_spanish:
                 spanish_events.append((event, web_event))
             else:
                 english_events.append((event, web_event))
 
         for event, web_event in english_events:
-            
-            matches = [spanish_web_event['Audio']
-                       for spanish_event, spanish_web_event
-                       in spanish_events
-                       if event.is_partner(spanish_event)]
-            if matches:
-                spanish_audio, = matches
-                event['audio'] = [web_event['Audio'], spanish_audio]
-            else:
-                event['audio'] = [web_event['Audio']]
-                
+            english_audio = web_event['Audio']
+
+            if english_audio != 'Not\xa0available':
+                matches = [spanish_web_event['Audio']
+                           for spanish_event, spanish_web_event
+                           in spanish_events
+                           if event.is_partner(spanish_event)]
+
+                if matches:
+                    spanish_audio, = matches
+                    spanish_audio['Label'] = 'Audio (SAP)'
+                    event['audio'] = [english_audio, spanish_audio]
+                    continue
```

Comment:
I think it would be a lot clear if you used an else here instead of this continue.

Reply:
Good point! How about https://github.com/opencivicdata/scrapers-us-municipal/pull/216/commits/b198f241a1eda6bec798dd72eaa9a2ac9a7367e3